packages/react/src/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { forwardRef } from 'react'
+import type { ForwardedRef, HTMLAttributes, PropsWithChildren } from 'react'
+import { VisuallyHidden } from '../VisuallyHidden'
+
+export type ErrorMessageProps = {
+  prefix?: string
+} & PropsWithChildren<HTMLAttributes<HTMLParagraphElement>>
+
+export const ErrorMessage = forwardRef(
+  (
+    { children, className, prefix = 'Invoerfout: ', ...restProps }: ErrorMessageProps,
```

Comment:
Should we render the colon ourselves to ensure the screen reader pauses between the prefix and the error message? (Assuming that it does.)

Reply:
Yeah, it adds a pause. Good idea to render it ourselves!